server.js

```diff
@@ -842,12 +903,14 @@ class Room {
     /**
      * Preparing room for the game
      */
-    gamePrepare() {
+    gamePrepare(prepareWords = true) {
         // changing state to 'play'
         this.state = "play";
 
-        // generating word list
-        this.freshWords = generateWords(this.settings);
+        if (prepareWords) {
```

Comment:
А давай чуть по-умнее:
```suggestion
        if (this.settings.wordsetType !== "playerWords") {
```
Тут же и аргумент `prepareWords` не будет нужен.